zulip_bots/zulip_bots/bots/converter/converter.py

```diff
@@ -72,8 +75,9 @@ def get_bot_converter_response(message, bot_handler):
                 results.append('`' + number + '` is not a valid number. ' + utils.QUICK_HELP)
                 continue
 
-            number = float(number)
-            number_res = copy.copy(number)
+            # cannot reassign "number" as a float after using as string, so changed name
+            convertNum = float(number)
```

Comment:
The variable `convertNum` is what you call camelCase, which violates [the PEP8 coding standard](https://www.python.org/dev/peps/pep-0008/). The variable should be `convert_num` instead! :)